public/electron-starter.js

```diff
@@ -3,13 +3,13 @@ const isDev = require('electron-is-dev');
 const path = require('path');
 const url = require('url');
 const electron = require('electron');
+
 const menuTemplate = require('./electronHelpers/menuTemplate');
 const runAutoUpdater = require('./electronHelpers/autoUpdater');
-const setupProtocols = require('./electronHelpers/setupProtocols');
 
 const { app, shell, Menu } = electron;
 
-require('electron-debug')({ enabled: true });
```

Comment:
would be good to leave this enabled so that the user can access the console for debugging